sote/game/scenes/game/inspectors/market.lua

```diff
@@ -6,7 +6,7 @@ local inspector = {}
 ---@return Rect
 local function get_main_panel()
 	local fs = ui.fullscreen()
-	local panel = fs:subrect(ut.BASE_HEIGHT * 2, ut.BASE_HEIGHT * 2, ut.BASE_HEIGHT * 40, ut.BASE_HEIGHT * 15, "left", 'up')
+	local panel = fs:subrect(ut.BASE_HEIGHT * 2, ut.BASE_HEIGHT * 2, ut.BASE_HEIGHT * 48, ut.BASE_HEIGHT * 15, "left", 'up')
```

Comment:
Can we use ' for strings from now on? We'll need a linter to make checking this consistent but it also makes the repo seem low quality (I know this is largely my fault too)